lgsm/config-default/config-lgsm/ckserver/_default.cfg

```diff
@@ -0,0 +1,183 @@
+##################################
+######## Default Settings ########
+##################################
+# DO NOT EDIT, ANY CHANGES WILL BE OVERWRITTEN!
+# Copy settings from here and use them in either:
+# common.cfg - applies settings to every instance.
+# [instance].cfg - applies settings to a specific instance.
+
+#### Game Server Settings ####
+
+## Server Parameters | https://docs.linuxgsm.com/configuration/start-parameters#additional-parameters
+# Edit with care | https://colonysurvival.gamepedia.com/Dedicated_Server#Installation_.28Linux.29
```

Comment:
This line still references colony survival and should probably be updated or removed accordingly.

Reply:
Thanks, will address this with the next rebase